modules/statistics/statistics.js

```diff
@@ -157,16 +160,17 @@ Statistics.init = function(options) {
  * @param {StatisticsOptions} options - The options to use creating the
  * Statistics.
  */
-export default function Statistics(xmpp, options) {
+export default function Statistics(conference, config) {
```

Comment:
Update the jsdoc or keep using options pl.

Reply:
updated